src/renderer/stores/game.ts

```diff
@@ -43,26 +43,18 @@ export default class GameStore {
   }
 
   @action
-  startGame = (privateKey: string) => {
+  startGame = async (privateKey: string) => {
```

Comment:
I couldn't find a call to `startGame` using Promise or `await`. Is this change intended?

Reply:
It seems previous work remain. I'll delete it.

Reply:
It seems previous work remain. I'll delete it.